src/content/en/fundamentals/performance/resource-prioritization.md

```diff
@@ -0,0 +1,351 @@
+  --aspect-ratio-h: 1;
+
+  position: relative;
+  max-width: 100%;
+  margin-bottom: 1ex;
+}
+
+.aspect-ratio > *:first-child {
+  width: 100%;
+}
+
+@supports (--custom-props: "true") {
+  .aspect-ratio::before {
+    display: block;
+    padding-top: calc(var(--aspect-ratio-h, 1) /
+        var(--aspect-ratio-w, 1) * 100%);
+    content: "";
+  }
+
+  .aspect-ratio > *:first-child {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+  }
+}
+</style>
+
+# Resource Prioritization – Getting the Browser to Help You {: .page-title }
+
+{% include "web/_shared/contributors/sgomes.html" %}
+
+Not every byte that is sent down the wire to the browser has the same degree of
+importance, and the browser knows this. Browsers have heuristics that attempt to
+make a best-guess at the most important resources to load first — such
+as CSS before scripts and images.
+
+That said, as with any heuristic, it doesn’t always work out; the browser might
+make the wrong decision, usually because it doesn’t have enough information at
+that time. This article explains the various tools you have at your disposal to
+influence the priority of content adequately in modern browsers, by letting them
+know what you’ll be needing later.
+
+## Default Priorities in the Browser
+
+As mentioned before, the browser assigns different relative priorities to
+different types of resources based on how critical they might be. So, for
+example, a `<script>` tag in your page’s `<head>` would be loaded in Chrome at a
+**High** priority (below CSS, at **Highest**), but that priority would change to
+**Low** if it has the async attribute (meaning it can be loaded and run
+asynchronously).
+
+Priorities become important when investigating loading performance in your site,
+so beyond the usual techniques of
+[measuring](/web/fundamentals/performance/critical-rendering-path/measure-crp)
+and
+[analyzing the critical rendering path](/web/fundamentals/performance/critical-rendering-path/analyzing-crp),
+it’s useful to know Chrome’s priority for each resource. You can find that in
+the Network panel in Chrome Developer Tools. Here’s what it looks like:
+
+
+<figure>
+  <div class="aspect-ratio"
+       style="width: 1810px; --aspect-ratio-w: 1810; --aspect-ratio-h: 564">
+    <img src="images/res-prio-priorities.png"
+    alt="An example of how priorities are displayed in Chrome Developer Tools">
+  </div>
+  <figcaption><b>Figure 1</b>: Priorities in Chrome Developer Tools. You may
+  need to enable the Priority column by right-clicking on the column headers.
+  </figcaption>
+</figure>
+
+
+These priorities will give you an idea of how much relative importance the
+browser attributes to each resource. And remember that subtle differences are
+enough for the browser to assign a different priority; for example, an image
+that is part of the initial render will be prioritized higher than an image that
```

Comment:
Remove future tense again: "... initial render is prioritized higher..."

Reply:
Bad habit, apparently :) Done.